server.js

```diff
@@ -77,7 +65,7 @@ app.post('api/login',function(req,res) {
   function(req, res) {
     // If this function gets called, authentication was successful.
     // `req.user` contains the authenticated user.
-    res.redirect('/users/' + req.user.username);
+    res.redirect('/users/' + req.user._id);
```

Comment:
You don't have a username field on the user model. I assume you will return the user to their profile page. But you must edit this redirect to send the user to the appropriate route!